tox.ini

```diff
@@ -6,6 +6,7 @@ skip_missing_interpreters=true
 deps = 
     pyshp
     pytest
+    
```

Comment:
I added numpy at one point because tox was throwing up a weird error that I was trying to debug and I got desperate. I removed numpy after I figured out the real culprit.